hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/systemcontracts/has/HasCallFactory.java

```diff
@@ -72,7 +72,8 @@ public HasCallFactory(
             @NonNull final MessageFrame frame) {
         requireNonNull(input);
         requireNonNull(frame);
-        final var enhancement = proxyUpdaterFor(frame).enhancement();
+        final var proxyUpdater = proxyUpdaterFor(frame);
```

Comment:
this change seems unnecessary?

Reply:
yes, leftover from something I reverted when I merged your update